### PR TITLE
Feature-6-DOC-1584 - Prepare TinyMCE 6.0 docs for automated deployments to staging and production

### PR DESCRIPTION
**Pull request by tinyamit**
Related Ticket: https://ephocks.atlassian.net/browse/DOC-1584

Description of Changes:
* RE pull request on previous version with requested changes

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [n/a] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [n/a] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: .github/workflows/feature_6_docs.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: Develop > Tiny docs 6.0
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - '*'
+    branches:
+      - 'feature/6**'
```
Comment (lnewson) [inline .github/workflows/feature_6_docs.yml:10]: Should this be something like this instead?
```suggestion
      - 'feature/6/**'
```

File: .github/workflows/feature_6_docs.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      run: sudo yarn global add @antora/cli@3.0.1
+
+    - name: (Feature - Docs 6.0) Install dependencies
+      run: yarn install
+
+    - name: (Feature - Docs 6.0) Build API References
+      run: yarn build
+
+    - name: (Feature - Docs 6.0) Build Website
+      run: antora ./antora-playbook-dev.yml
```
Comment (lnewson) [inline .github/workflows/feature_6_docs.yml:40]:
```suggestion
      run: yarn antora ./antora-playbook-dev.yml
```

Looks like this was missed from the previous PR? If we change it to this we don't need the global version installed above.

File: .github/workflows/release_6_docs.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      run: sudo yarn global add @antora/cli@3.0.1
+
+    - name: (Release - Docs 6.0) Install dependencies
+      run: yarn install
+
+    - name: (Release - Docs 6.0) Build API References
+      run: yarn build
+
+    - name: (Release - Docs 6.0) Build Website
+      run: antora ./antora-playbook-staging.yml
```
Comment (lnewson) [inline .github/workflows/release_6_docs.yml:40]:
```suggestion
      run: yarn antora ./antora-playbook-staging.yml
```

File: .github/workflows/feature_6_docs.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        cache: 'yarn'
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Antora@3.0.1
+      run: sudo yarn global add @antora/cli@3.0.1
+
```
Comment (lnewson) [inline .github/workflows/feature_6_docs.yml:32]: Why is this needed? As we should be able to use the version of antora installed in the next step?
```suggestion
```

File: .github/workflows/release_6_docs.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        cache: 'yarn'
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Antora@3.0.1
+      run: sudo yarn global add @antora/cli@3.0.1
+
```
Comment (lnewson) [inline .github/workflows/release_6_docs.yml:32]:
```suggestion
```

File: .github/workflows/staging_6_docs.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        cache: 'yarn'
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Antora@3.0.1
+      run: sudo yarn global add @antora/cli@3.0.1
+
```
Comment (lnewson) [inline .github/workflows/staging_6_docs.yml:32]:
```suggestion
```

File: .github/workflows/staging_6_docs.yml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    - name: (Staging - Docs 6.0) Install dependencies
+      run: yarn install
+
+    - name: (Staging - Docs 6.0) Build API References
+      run: yarn build
+
+    - name: (Staging - Docs 6.0) Build Website
+      run: antora ./antora-playbook-staging.yml
```
Comment (lnewson) [inline .github/workflows/staging_6_docs.yml:40]:
```suggestion
      run: yarn antora ./antora-playbook-staging.yml
```

**Review by lnewson — APPROVED** [review]

File: .github/workflows/feature_6_docs.yml (hunk 1 above)
Comment (tinyamit) [reply .github/workflows/feature_6_docs.yml:10]: Its to give people some room to have minor version if they want to. What do you think should we @lnewson ?

Comment (lnewson) [reply .github/workflows/feature_6_docs.yml:10]: Ahh fair enough... I guess I was just a little concerned about things like `feature/6-DOC-1234` being matched, but it's probably not something to worry about.

File: .github/workflows/feature_6_docs.yml (hunk 2 above)
Comment (tinyamit) [reply .github/workflows/feature_6_docs.yml:40]: Yeah I missed it, fixed now !

File: .github/workflows/feature_6_docs.yml (hunk 4 above)
Comment (tinyamit) [reply .github/workflows/feature_6_docs.yml:32]: Yeah I missed it, fixed now !
